.travis.yml

```diff
@@ -10,22 +10,28 @@ cache:
   directories:
   - '$HOME/.m2/repository'
 
-addons:
-  chrome: stable
-
 stages:
 - build
 - test
+- system-test
 
 jobs:
   include:
   - stage: build
     script: mvn clean install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
   - stage: test
-    script: mvn test -B -V
-  - stage: test
+    script:
+    - mvn org.jacoco:jacoco-maven-plugin:prepare-agent test org.jacoco:jacoco-maven-plugin:report -Dmaven.test.failure.ignore=false -B -V
```

Comment:
what is the purpose of `-Dmaven.test.failure.ignore=false`? isn't it the default value?

Reply:
removed